frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -218,6 +218,7 @@ function ReaderBookmark:onShowBookmark()
         width = Screen:getWidth(),
         height = Screen:getHeight(),
         cface = Font:getFace("x_smallinfofont"),
+        line_color = require("ffi/blitbuffer").COLOR_WHITE,
```

Comment:
May be you need `perpage `here (like you have it below in readertoc). Otherwise, if you change item_per_page via spin widget, it's not updated till next koreader restart.